samples/maps/geo-map/marker-layouts/src/MapMarkerLayouts.tsx

```diff
@@ -0,0 +1,199 @@
+
+                    </select>
+                </div>
+
+                <div className="igComponent" >
+                    <IgrGeographicMap
+                        ref={this.onMapRef}
+                        width="100%"
+                        height="100%"
+                        zoomable="true" />
+                </div>
+
+
+                <div className="igOverlay-bottom-right">Imagery Tiles: @OpenStreetMap</div>
+            </div>
+        );
+    }
+
+    public onSelectionModeChange = (e: any) => {
+        this.setState({ selectionCollisionType: e.target.value });
+        this.geoSeries.markerCollisionAvoidance = e.target.value;
+    }
+
+    public onMapRef(geoMap: IgrGeographicMap) {
+        if (!geoMap) { return; }
+
+        this.geoMap = geoMap;
+        // hiding OpenStreetMap
+        this.geoMap.backgroundContent = undefined;
+        // zooming map to region of North America
+        this.geoMap.windowRect = { left: 0.2, top: 0.1, width: 0.6, height: 0.6 };
+
+        // loading a shapefile with geographic shapes
+        const sds = new IgrShapeDataSource();
+        sds.importCompleted = this.onDataLoaded;
+        sds.shapefileSource = "https://static.infragistics.com/xplatform/shapes/world_countries_all.shp";
+        sds.databaseSource  = "https://static.infragistics.com/xplatform/shapes/world_countries_all.dbf";
+        sds.dataBind();
+    }
+
+    public onDataLoaded(sds: IgrShapeDataSource, e: any) {
+        const shapeRecords = sds.getPointData();
+        console.log("loaded WorldCountries.shp " + shapeRecords.length);
+
+        const countries: any[] = [];
+
+
+        for (const record of shapeRecords) {
+            // using field/column names from .DBF file
+            const country = {
+                points: record.points,
+                name: record.fieldValues.Name,
+                org: record.fieldValues.Alliance,
+                pop: record.fieldValues.Population
+            };
+
+            countries.push(country);
+
+        }
+
+        this.createSeries(countries, "rgba(203, 205, 204, 1)");
+
+    }
+
+    public createSeries(shapeData: any[], shapeBrush: string) {
+        const seriesName = "series";
+        this.geoSeries = new IgrGeographicShapeSeries({ name: seriesName });
+        this.geoSeries.dataSource = shapeData;
+        this.geoSeries.shapeMemberPath = "points";
+        this.geoSeries.brush = "lightgrey";
+        this.geoSeries.outline = "Black";
+        this.geoSeries.tooltipTemplate = this.createTooltip;
+        this.geoSeries.thickness = 1;
+        this.geoSeries.markerType = MarkerType.Hexagon;
+        this.geoSeries.markerCollisionAvoidance = CollisionAvoidanceType.Fade;
+
+        this.geoSeries.markerBrush = "DodgerBlue";
+        this.geoSeries.markerOutline = "DodgerBlue";
+        this.geoSeries.markerTemplate = this.getMarker();
+
+        this.geoMap.series.add(this.geoSeries);
+    }
+
+    public getMarker(): any {
```

Comment:
you should replace this method with getMarker() from this [sample](https://github.com/IgniteUI/igniteui-react-examples/blob/master/samples/maps/geo-map/marker-template/src/MapMarkerTemplate.tsx#L56) and then display :name" instead of "pop" on this [line ](https://github.com/IgniteUI/igniteui-react-examples/blob/master/samples/maps/geo-map/marker-template/src/MapMarkerTemplate.tsx#L69)